packages/gatsby-plugin-sharp/src/index.js

```diff
@@ -95,8 +96,12 @@ const generalArgs = {
   sizeByPixelDensity: false,
 }
 
-const healOptions = (args, defaultArgs) => {
-  let options = _.defaults({}, args, defaultArgs, generalArgs)
+const healOptions = (pluginOptions, args, defaultArgs) => {
+  const quality =
+    pluginOptions.defaultQuality && args.quality === 50
```

Comment:
Don't really like this, but because `gatsby-transformer-sharp` has a default `arg` of `50`, you will never get the `defaultQuality` option applied.  I could update the PR to remove the default `arg` from `gatsby-transformer-sharp`.  Then this would be simplified to just: 

```javascript
let options = _.defaults({}, {...pluginOptions}, args, defaultArgs, generalArgs);
```